### DISCUSSION TITLE
nvim-tree + .gitignore: one directory is empty but visible

### DISCUSSION THREAD

**Discussion by wojcraft** [nvim-tree/nvim-tree.lua#2592] [Q&A]
Hello,

Below I uploaded the screenshot of an empty `.config` directory listed in nvim-tree. In normal `ls -lah` it is not empty, but nvim-tree + .gitignore makes it - I don't know why. I'm here asking for help, to see my files and directories in the `.config` dir through nvim-tree. If you need more context, I want you to know I'm here to give it. Have a nice day or night!

![image](https://github.com/nvim-tree/nvim-tree.lua/assets/141335133/e18683a1-bdaf-4072-88a0-6c5a50ec8398)

**Comment by alex-courtis** [comment]
`*` matches anything but slash i.e. directories. Maybe try `*/`.

It looks like you are trying to version control your home directory or dotfiles. That never ends well and is generally considered unix/git bad practice

See #2142 #1966

Many solutions here: https://wiki.archlinux.org/title/Dotfiles
